src/panels/config/cloud/login/cloud-login.ts

```diff
@@ -114,6 +114,7 @@ export class CloudLogin extends LitElement {
                   )}
                   id="email"
                   type="email"
+                  autocomplete="username"
```

Comment:
Since this is another login then the login used for ha, and it is on the same hostname, will the autocomplete now suggest the input of the ha login here and vice versa? Maybe better to leave it out here in that case?

Reply:
It's a possibility that the password manager could suggest both, but it's up to the user and the password manager to sort out the URL differences and pick the appropriate credentials. We shouldn't take away that possibility.

I personally use Bitwarden and this would be easy to sort out by requiring both the path and domain to match in the URL.